lib/ui/painting/picture.cc

```diff
@@ -81,8 +82,8 @@ Dart_Handle Picture::RasterizeToImage(sk_sp<SkPicture> picture,
   }
 
   auto* dart_state = UIDartState::Current();
-  tonic::DartPersistentValue* image_callback =
-      new tonic::DartPersistentValue(dart_state, raw_image_callback);
+  auto image_callback = std::make_shared<tonic::DartPersistentValue>(
```

Comment:
I was thinking of using std::unique_ptr then using std::move semantics with it.  It will be quite a refactor though I realize since `ui_task` is getting copied.  Thats why `std::shared_ptr` is convenient.